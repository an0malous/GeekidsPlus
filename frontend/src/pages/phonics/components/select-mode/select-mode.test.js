import React from 'react'
import renderer from 'react-test-renderer';
import SelectMode from './select-mode.component';

describe('AlphabetCard Component', () => {
   let component;

   beforeEach(() => {
     component = renderer
     .create(<SelectMode />)
     .toJSON()
   });
   
   it('Renders', () => {
      expect(component).toMatchSnapshot()
   })
});

