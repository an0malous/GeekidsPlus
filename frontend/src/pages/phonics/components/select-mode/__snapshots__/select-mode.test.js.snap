// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlphabetCard Component Renders 1`] = `
<div
  className="ui container"
>
  <div
    className="ui container"
  >
    <div
      className="ui center aligned grid"
    >
      <div
        className="column"
      >
        <div
          onClick={null}
        >
          ＜
        </div>
      </div>
      <div
        className="column"
      >
        <div
          onClick={[Function]}
        >
          a
        </div>
      </div>
      <div
        className="column"
      >
        <div
          onClick={[Function]}
        >
          ＞
        </div>
      </div>
    </div>
  </div>
</div>
`;
